packages/blocks/src/components/atoms/HoverCard/HoverCard.tsx

```diff
@@ -0,0 +1,68 @@
+import { flip, offset as offsetFloating, safePolygon, shift, useFloating, useFocus, useHover, useInteractions } from '@floating-ui/react';
+import cx from 'classnames';
+import React, { useState } from 'react';
+
+type BaseProps = {
+  children?: React.ReactNode | string
+  className?: string
+  placement?: 'top' | 'top-start' | 'top-end' | 'right' | 'right-start' | 'right-end' | 'bottom' | 'bottom-start' | 'bottom-end' | 'left' | 'left-start' | 'left-end'
+  overlay: (close: () => void) => React.ReactNode
+  offset?: [number, number] // default [skidding, distance]
+  delay?: number,
+  disableFit?: boolean
+}
+
+export const HoverCard = ({
+  children,
+  className,
+  placement = 'right-start',
+  overlay,
+  offset = [0, 10],
```

Comment:
Maybe we could have a smaller offset of `6` by default so the card and trigger feel more connected?